os/binfmt/binfmt_loadbinary.c

```diff
@@ -206,8 +204,12 @@ int load_binary(int binary_idx, FAR const char *filename, load_attr_t *load_attr
 
 	BIN_ID(binary_idx) = pid;
 	BIN_STATE(binary_idx) = BINARY_LOADING_DONE;
+	if (load_attr->priority >= FAULTMSGSENDER_PRIORITY) {
+		BIN_RTTYPE(binary_idx) = BINARY_TYPE_REALTIME;
+	} else {
+		BIN_RTTYPE(binary_idx) = BINARY_TYPE_NONREALTIME;
+	}
```

Comment:
When main task is not realtime, but sub task is a realtime, is this ok?

Reply:
Yes. I think most of usages are same with what you said.